fastly/resource_fastly_alert.go

```diff
@@ -318,7 +318,9 @@ func buildEvaluationStrategy(v map[string]any) map[string]any {
 
 	// Optional attributes
 	if value, ok := v["ignore_below"]; ok {
-		m["ignore_below"] = value.(float64)
+		if value.(float64) > 0 {
```

Comment:
Yeah we've had to work around these sort of 'quirks' before, for example:
https://github.com/fastly/terraform-provider-fastly/blob/ef22e2c5b2143ba5c5d48a92eaaec79f713af21a/fastly/block_fastly_service_logging_cloudfiles.go#L387-L403

Reply:
Also, rather than doing the assertion twice try this...

```go
if v := value.(float64); v > 0 {
    m["ignore_below"] = v
}
```